docs/conf.py

```diff
@@ -10,88 +10,90 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
+import os
+import sys
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('.'))
-sys.path.insert(0, os.path.abspath('..'))
-sys.path.insert(0, os.path.abspath('../crispy_forms'))
-sys.path.insert(0, os.path.abspath('../crispy_forms/templatetags'))
-sys.path.append(os.path.abspath('_themes'))
+sys.path.insert(0, os.path.abspath("."))
+sys.path.insert(0, os.path.abspath(".."))
+sys.path.insert(0, os.path.abspath("../crispy_forms"))
+sys.path.insert(0, os.path.abspath("../crispy_forms/templatetags"))
+sys.path.append(os.path.abspath("_themes"))
 
 from django.conf import settings
+
 settings.configure()
 
 
 # -- General configuration -----------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode']
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.viewcode"]
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = ["_templates"]
 
 # The suffix of source filenames.
-source_suffix = '.rst'
+source_suffix = ".rst"
 
 # The encoding of source files.
-#source_encoding = 'utf-8'
+# source_encoding = 'utf-8'
 
 # The master toctree document.
-master_doc = 'index'
+master_doc = "index"
 
 # General information about the project.
-project = 'django-crispy-forms'
-copyright = '2009-2021, Miguel Araujo, Daniel Feldroy and contributors'
+project = "django-crispy-forms"
+copyright = "2009-2021, Miguel Araujo, Daniel Feldroy and contributors"
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '1.11'
+version = "1.11"
 # The full version, including alpha/beta/rc tags.
-release = '1.11.0'
+release = "1.11.1"
```

Comment:
This is the line that matters here. Everything else in this file should be formatting changes only